Libraries/LibGfx/Painter.cpp

```diff
@@ -1086,6 +1268,124 @@ void Painter::draw_text(const Rect& rect, const Utf32View& text, const Font& fon
     }
 }
 
+void Painter::draw_text(const Rect& rect, const StringView& raw_text, const TTF::ScaledFont& font, TextAlignment alignment, Color color, TextElision elision)
```

Comment:
Same duplication issue here. We need to find a way to share all of it between different font types.

Let's rearchitect the `Gfx::Font` class to be a wrapper around either a simple bitmap font (the system format) or a TrueType font. Then we can have a font-type-agnostic API in `Painter`.